### DISCUSSION TITLE
Indicator for NetworkCollision types

### DISCUSSION THREAD

**Discussion by maxkratt** [FirstGearGames/FishNet#648] [Feature Request]
Request by Awesome_AA: [Discord Link](https://discord.com/channels/424284635074134018/1227853758147596369/1227853758147596369)
> Request for Indicator for NetworkCollision types.
> It's really simple and easy to implement (I already modified the script), but also pass the type of NetworkCollision which has happened > (OnPostPhysicsTransformSync, OnPostPhysicsSimulation, or OnPostReplicateReplay) when invoking OnEnter/OnExit/OnStay.

> This is useful if, for example, we want to play a sound once when a player hits something; instead of 10-15 times due to reconcile OnEnters.

**Comment by FirstGearGames** [comment]
This would require changing the events API, which won't be something we'll be doing given it's out of experimental.

But also, even if I was open to changing the API I do not feel this is necessary given the developer can just check **PredictionManager.IsReconciling**.
